model/task_lifecycle.go

```diff
@@ -2131,7 +2144,7 @@ func MarkTasksReset(ctx context.Context, taskIds []string, caller string) error
 
 	catcher := grip.NewBasicCatcher()
 	for _, t := range tasks {
-		catcher.Wrapf(UpdateUnblockedDependencies(ctx, &t), "clearing unattainable dependencies for task '%s'", t.Id)
+		catcher.Wrapf(UpdateUnblockedDependencies(ctx, []task.Task{t}), "clearing unattainable dependencies for task '%s'", t.Id)
```

Comment:
Can this just pass in the entire slice of tasks rather than call it once per task?